storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -427,36 +491,93 @@ Options
+	- The common part for InnoDB table spaces.
+      * - `--innodb-directories`
+        - A list of directories 'dir1;dir2;..;dirN' to scan for tablespace
+          files. Default is to scan the directories specified via the
+          `--innodb-data-home-dir`, `--innodb-undo-directory`, and
+          :option:`--datadir` options.
+      * - `--innodb-doublewrite-file`
+      * - `--innodb-doublewrite`
+      * - `--innodb-extra-undoslots`
+      * - `--innodb-fast-checksum`
+      * - `--innodb-file-io-threads`
+	- The number of file I/O threads in InnoDB.
+      * - `--innodb-file-per-table`
+	- Stores each InnoDB table to an :file:`.ibd` file in the database dir.
+      * - `--innodb-flush-log-at-trx-commit`
+	- Set to 0 (write and flush once per second), 1 (write and flush at each
+          commit) or 2 (write at commit, flush once per second).
+      * - `--innodb-flush-method`
+	- With which method to flush data.
+      * - `--innodb-force-recovery`
+	- Helps to save your data in case the disk image of the database becomes
+          corrupt.
+      * - `--innodb-io-capacity`
+	- Number of IOPs the server can do. Tunes the background IO rate
+      * - `--innodb-lock-wait-timeout`
+      * - `--innodb-log-block-size`
+	- The log block size of the transaction log file. Changing for created
+          log file is not supported. Use on your own risk!
+      * - `--innodb-log-buffer-size`
+	- The size of the buffer which InnoDB uses to write log to the log files
+          on disk.
+      * -  `--innodb-log-checksums`
+        - Whether to compute and require checksums for InnoDB redo
+          log blocks. Defaults to on; use `--skip-innodb-log-checksums` to disable.
+      * - `--innodb-log-files-in-group`
+	- The number of log files in the log group. InnoDB writes to the files
+          in a circular fashion. Value 3 is recommended here.
+      * - `--innodb-log-file-size`
+	- The size of each log file in a log group.
+      * - `--innodb-log-group-home-dir`
+	- The path to InnoDB log files.
+      * - `--innodb-max-dirty-pages-pct`
+	- The percentage of dirty pages allowed in bufferpool.
+      * - `--innodb-open-files`
+	- How many files at the maximum InnoDB keeps open at the same time.
+      * - `--innodb-page-size`
+	- The universal page size of the database.
+      * - `--innodb-read-io-threads`
+	- The number of background read I/O threads in InnoDB.
+      * -  `--innodb-redo-log-encrypt`
+	- Enable or disable Encryption of REDO tablespace.
+      * - `--innodb-undo-directory`
+	- Directory where undo tablespace files live, this path can
+          be absolute.
+      * - `--innodb-undo-log-encrypt`
+	- Enable or disable Encrypt of UNDO tablespace.
+      * - `--innodb-undo-tablespaces` 
+	- The number of undo tablespaces to use.
+      * - `--innodb-use-native-aio`
+	- Use native AIO if supported on this platform.
+      * - `--innodb-write-io-threads`
+	- The number of background write I/O threads in InnoDB.
```

Comment:
i don't think we need to provide description for each InnoDB option, should be enough if we mention that these are recognized by xtrabackup

Reply:
Under --innodb-miscellaneous there was a list of --innodb-* options that xtrabackup supports. Does it make sense to keep this list or shall i remove it altogether?

Reply:
I think it is better to keep the list. But I don't think it makes sense to duplicate option descriptions which can be found in mysql docs